### PR TITLE
Add Support Bundle "Download Logs" docs

### PR DESCRIPTION
**Pull request by alexanderahn**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208809919368871

### CODE REVIEW

File: docs/architecture/deployment/private-saas/download-logs.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+You can use the Download Logs feature to capture logs using the Services and time selectors.
+
+- All services are selected by default.
+- The download supports one hour of logs from the Start Time.
+
+The captured logs include all relevant Kubernetes services, including sandbox pods.
+
+- Configuration Data Collection:
+
+  - Captures Kubernetes cluster configuration, such as resource quotas and node configuration.
```
Comment (georgekuruvillak) [inline docs/architecture/deployment/private-saas/download-logs.md:29]: we capture prophecy configuration i.e. prophecy CR and config map.

File: docs/architecture/deployment/private-saas/download-logs.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  - Collects Helm charts, deployment manifests, flags, and other configuration files.
```
Comment (georgekuruvillak) [inline docs/architecture/deployment/private-saas/download-logs.md:30]: only configuration files.

File: docs/architecture/deployment/private-saas/download-logs.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  - Includes current versions of deployed services and Kubernetes components.
+
+- Environment State Capture:
+  - Captures pod status, node health checks, and resource allocation across the cluster, such as kubectl describe nodes and kubectl describe pods.
```
Comment (georgekuruvillak) [inline docs/architecture/deployment/private-saas/download-logs.md:34]: we dont do that in this feature. for that we have health check and monitoring.

**Review by georgekuruvillak — CHANGES_REQUESTED** [review]
I have added what is in this now. cc: @manjunathprophecy

File: docs/architecture/deployment/private-saas/download-logs.md (hunk 1 above)
Comment (alexanderahn) [reply docs/architecture/deployment/private-saas/download-logs.md:29]: Done.

File: docs/architecture/deployment/private-saas/download-logs.md (hunk 2 above)
Comment (alexanderahn) [reply docs/architecture/deployment/private-saas/download-logs.md:30]: Done.

File: docs/architecture/deployment/private-saas/download-logs.md (hunk 3 above)
Comment (alexanderahn) [reply docs/architecture/deployment/private-saas/download-logs.md:34]: Done.

**Comment by alexanderahn** [comment]
To update order of other files before merge.

File: docs/architecture/deployment/private-saas/download-logs.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: Download Logs
+id: download-logs
+description: Download Prophecy support logs
+sidebar_position: 3
+tags:
```
Comment (manjunathprophecy) [inline docs/architecture/deployment/private-saas/download-logs.md:6]: we have some pre-requisite for enabling this feature for private SAAS: documented here : https://coda.io/d/Cloud_dptsTdZM0LD/Setting-up-prophecy-log-download-options_suDxgcHw#_luDrg_xx

Comment (alexanderahn) [reply docs/architecture/deployment/private-saas/download-logs.md:6]: Done.

**Pull request merged**